Doc/whatsnew/3.9.rst

```diff
@@ -202,6 +202,16 @@ now raises :exc:`ImportError` instead of :exc:`ValueError` for invalid relative
 import attempts.
 (Contributed by Ngalim Siregar in :issue:`37444`.)
 
+ipaddress
+---------
+
+:mod:`ipaddress` now supports IPv6 Scoped Addresses (IPv6 address with suffix ``%<scope_id>``).
+
+Scoped IPv6 could be parsed to :class:`ipaddress.IPv6Address` constructor.
```

Comment:
I would recommend making some minor grammar changes: 

1) "could be" should instead be "can be" 
2) I don't think it's necessary to specify "constructor", that's typically implied when referring to a specific class. 

```suggestion
Scoped IPv6 addresses can be parsed using :class:`ipaddress.IPv6Address`.
```